packages/react-positioning/src/usePopper.ts

```diff
@@ -1,291 +1,37 @@
-import { useEventCallback, useIsomorphicLayoutEffect, useFirstMount, canUseDOM } from '@fluentui/react-utilities';
+import {
```

Comment:
there is no popper anymore. We should rename this hook file and he hook as well for consistency or am I missing something? ty

Reply:
yeah, this is not public (i.e. not reexported in react-components) I don't mind renaming it to `usePositioning` ?

Reply:
Done in [4fae144](https://github.com/microsoft/fluentui/pull/21828/commits/4fae144a4a527f3685dd27c9b02e0f20e5b255a8)